pkgs/development/tools/glpaper/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchhg
+, meson, ninja, pkgconfig
+, wlroots, wayland, wayland-protocols
+, pixman, libxkbcommon
+,libX11
+, libGL, libglvnd
+}:
+
+stdenv.mkDerivation {
+  name = "glpaper";
+  version = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/glpaper";
+    rev = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+    sha256 = "04y12910wvhy4aqx2sa63dy9l6nbs7b77yqpdhc96x2b3mgzgjfs";
+  };
+
+  nativeBuildInputs = [ meson ninja pkgconfig ];
+  buildInputs = [
+    wlroots wayland wayland-protocols wlroots
+    pixman libxkbcommon
+    libX11
+    libGL libglvnd
+  ];
+
+  enableParallelBuilding = true;
```

Comment:
`enableParallelBuilding` is enabled by default in meson and cmake packages.

Reply:
Done.